app/routes/interestgroups/components/InterestGroup.tsx

```diff
@@ -1,52 +1,40 @@
 import { Flex } from '@webkom/lego-bricks';
+import cx from 'classnames';
 import { Link } from 'react-router-dom';
 import { Image } from 'app/components/Image';
 import styles from './InterestGroup.css';
 import type { Group } from 'app/models';
 // TODO: rather handle this in the backend
 const SAMPLE_LOGO = 'https://i.imgur.com/Is9VKjb.jpg';
```

Comment:
Dafaq is this thing? It's not your doing so leave it if you will - I just couldn't not comment on it.. Wouldn't it be a lot better to use the abakus-logo with just the red ball for interest groups that are missing logos than some random imgur image?

Reply:
Yes, I agree, it does not show even when the normal logos does not load in, so I did not think of it, but I will change it!